plugins/source/aws/resources/services/inspector2/findings.go

```diff
@@ -48,8 +48,20 @@ The 'request_account_id' and 'request_region' columns are added to show from whe
 func fetchInspector2Findings(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
 	c := meta.(*client.Client)
 	svc := c.Services().Inspector2
-	input := inspector2.ListFindingsInput{MaxResults: aws.Int32(100)}
-	paginator := inspector2.NewListFindingsPaginator(svc, &input)
+	var err error
+	input := &inspector2.ListFindingsInput{}
+	if c.Spec.TableOptions.Inspector2Findings != nil {
+		input, err = c.Spec.TableOptions.Inspector2Findings.ListFindings()
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
Would it be possible to 'hold onto' defaults here? Something like `if input.Maxresults == 0 { input.MaxResults = aws.Int32(100) }`
I understand it wouldn't work for values with a valid zero value

Reply:
If we want to do that I would suggest that we return an error when calling `ListFindings()`